### PR TITLE
Trade Settings Rework

### PR DESCRIPTION
**Pull request by Cuphat**
"Shuffle Weird Egg" and "Skip Child Zelda" have been combined into a single setting "Shuffle Child Trade Item". These settings were mutually exclusive, so no flexibility is lost.

Combines "Adult Trade Sequence Earliest Item" and "Adult Trade Sequence Latest Item" into a single, multiple select setting "Adult Trade Sequence Items".

### CODE REVIEW

**Comment by mracsys** [comment]
I just want to say thank you. This will make trade quest shuffle a lot easier to add.

File: ItemPool.py [hunk 1]
```diff
@@ -210,16 +210,16 @@
...
+    ("Pocket Egg",   "Pocket Egg"),
+    ("Pocket Cucco", "Pocket Cucco"),
+    ("Cojiro",       "Cojiro"),
+    ("Odd Mushroom", "Odd Mushroom"),
+    ("Poachers Saw", "Poachers Saw"),
+    ("Broken Sword", "Broken Sword"),
+    ("Prescription", "Prescription"),
+    ("Eyeball Frog", "Eyeball Frog"),
+    ("Eyedrops",     "Eyedrops"),
+    ("Claim Check",  "Claim Check"),
```
Comment (cjohnson57) [inline ItemPool.py:222]: It doesn't seem necessary for this to be a dict anymore

**Review by cjohnson57 — CHANGES_REQUESTED** [review]
Just the one small thing, although is keeping the adult trade item start necessary once trade shuffle is on the table? Unsure

**Comment by fenhl** [comment]
Yes, the earliest and latest item setting still make sense if the trade shuffle setting is off. See [discussion in #dev-public-talk on August 22](https://discord.com/channels/274180765816848384/512048482677424138/1011063982993461258).

File: ItemPool.py (hunk 1 above)
Comment (Cuphat) [reply ItemPool.py]: Yeah, I was lazy at the time. It's no longer a dict now.

**Review by cjohnson57 — APPROVED** [review]

**Pull request merged**